app/services/bulk_upload/lettings/year2023/row_parser.rb

```diff
@@ -1606,4 +1607,8 @@ def first_time_property_let_as_social_housing
   def is_carehome
     field_127.present? ? 1 : 0
   end
+
+  def downcase(sentence)
+    downcase_first_letter(sentence)
+  end
```

Comment:
Add a more concisely named private method to each row parser, using the downcase_first_letter method from the helper.

Reply:
Renamed the parameter to text from sentence